webserver/jersey/src/main/java/io/helidon/webserver/jersey/HelidonHK2InjectionManagerFactory.java

```diff
@@ -67,6 +67,9 @@ public InjectionManager create(Object parent) {
             result = new HelidonInjectionManager(forApplication, wrapper.injectionManager, wrapper.application);
             LOGGER.finest(() -> "Creating injection manager " + forApplication + " with shared "
                     + wrapper.injectionManager);
+        } else if (parent instanceof HelidonInjectionManager) {
+            result = (InjectionManager) parent;
+            LOGGER.finest(() -> "Using injection manager " + result);
```

Comment:
Duplicates lines 62–63 exactly; was that your intent? If so, maybe add this `if` condition as an `||` on line 61?

Reply:
No intent, was just being lazy. Will fix.

Reply:
Actually, I will update the log messages line and keep them separate as it separates the cases nicely